src/Nethermind/Nethermind.Evm/TransactionProcessor.cs

```diff
@@ -267,6 +259,11 @@ private void Execute(Transaction transaction, BlockHeader block, ITxTracer txTra
 
                     substate = _virtualMachine.Run(state, txTracer);
                     unspentGas = state.GasAvailable;
+
+                    if (txTracer.IsTracingAccess)
+                    {
+                        txTracer.ReportAccess(state.AccessedAddresses, state.AccessedStorageCells);
```

Comment:
this should be inside the warm up call, otherwise you keep repeating the growing lists

Reply:
??? 

This is one call after VM excecution. This grabs the already built access list, nothing is growing here.